includes/class-edd-utilities.php

```diff
@@ -126,44 +134,65 @@ public function get_registry( $name ) {
 	/**
 	 * Retrieves a date format string based on a given short-hand format.
 	 *
+	 * @see edd_get_date_format()
+	 * @see edd_get_date_picker_format()
+	 *
 	 * @since 3.0
 	 *
-	 * @param string $format Shorthand date format string. Accepts 'date', 'time', 'mysql', or
-	 *                       'datetime'. If none of the accepted values, the original value will
-	 *                       simply be returned. Default is the value of the `$date_format` property,
-	 *                       derived from the core 'date_format' option.
+	 * @param string $format Shorthand date format string. Accepts 'date', 'time', 'mysql', 'datetime',
+	 *                       'picker-field' or 'picker-js'. If none of the accepted values, the
+	 *                       original value will simply be returned. Default is the value of the
+	 *                       `$date_format` property, derived from the core 'date_format' option.
 	 * @return string date_format()-compatible date format string.
 	 */
-	public function get_date_format_string( $format ) {
+	public function get_date_format_string( $format = 'date' ) {
 
+		// Default to 'date' if empty
 		if ( empty( $format ) ) {
 			$format = 'date';
 		}
 
-		if ( ! in_array( $format, array( 'date', 'time', 'datetime', 'mysql' ) ) ) {
+		// Bail if format is not known
+		if ( ! in_array( $format, array( 'date', 'time', 'datetime', 'mysql', 'date-attribute', 'date-js' ) ) ) {
 			return $format;
 		}
 
-		switch( $format ) {
-			case 'time':
-				$format = $this->get_time_format();
+		// What known format are we getting?
+		switch ( $format ) {
+
+			// jQuery UI Datepicker fields, placeholders, etc...
+			case 'date-attribute' :
+				$retval = 'yyyy-mm-dd';
```

Comment:
Purely a style question, but ... why the variable name change from something self-documenting (`$format`) to something that's indicative only of being a return value?